### ISSUE TITLE
The plugin just doesnt work on any Language and gives same error

### ISSUE THREAD

**Issue by shubham-cpp** [kkoomen/vim-doge#114]
**Describe the bug**

**Settings**

**To Reproduce**

- Create file `doge.vim` with:

```vim
set nocompatible
set runtimepath^=/path/to/vim-doge.nvim
filetype plugin indent on
syntax on
set hidden
```

- Start (neo)vim with command: `vim -u doge.vim`

Did the above steps and no errors occurred
I have also installed doxygen(from standard arch repo), numpy(also from std repo) and LDoc(from aur on arch linux)

**Expected behavior**

Pressing `<leader>d` should provide documentation but it doesnt on any language(tested on python,lua and cpp)

**Screenshots**

![problem ss](https://ibb.co/mDKzGkc)

**Output of `vim --version`**
NVIM v0.5.0-753-g4b00916e9
I'm using Nvim nighlty build

**Comment by jerri** [comment]
You probably missed the step, where you have to install dependencies? You have to `call doge#install()` to download the binary.
Be aware though, that there is an open problem right now with the linux-binary, which can't be executed right now. But at least after the install step, you should get an error message.
@kkoomen : Might make sense to catch the case when a user forgot to download the binary and output an appropriate message in that case. I also was wondering initially why it didn't work until I found the info about the install step.

**Comment by shubham-cpp** [comment]
I'm using vim-plug and I have this in my init.vim
`Plug 'kkoomen/vim-doge', { 'do': { -> doge#install() } }`

**Comment by shubham-cpp** [comment]
> you should get an error message.

Thats exactly what i got. I have uploaded the screenshot

**Comment by shubham-cpp** [comment]
> Be aware though, that there is an open problem right now with the linux-binary, which can't be executed right now. 

Is there any work around like manually installing the binaries(like some people do with you-complete-me)

**Comment by kkoomen** [comment]
> Might make sense to catch the case when a user forgot to download the binary and output an appropriate message in that case. I also was wondering initially why it didn't work until I found the info about the install step.

I can add that later, although it's not my problem if people don't read the docs carefully.

**Comment by kkoomen** [comment]
@shubham-cpp Please be patient as @jerri mentioned. I'm working on a fix for the builds at this point. You can checkout to v3.2.1 and run `call doge#install()` for now. This version is without binary.

**Comment by kkoomen** [comment]
@shubham-cpp Please try out the new version that I mentioned [here](https://github.com/kkoomen/vim-doge/issues/112#issuecomment-715531000) and let me know if it works or not.

**Comment by jerri** [comment]
Yay. For me it looks good. Tested with php and python and could created document-skeletons again. Thanks!

**Comment by kkoomen** [comment]
I've just released v3.4.0, a new stable version. I'll close this issue for now as 2 other people have confirmed that the Linux binary works now. Thanks for contributing.
